sdk/monitor/azure-monitor-query/azure/monitor/query/aio/_logs_query_client_async.py

```diff
@@ -34,15 +35,18 @@ class LogsQueryClient(object): # pylint: disable=client-accepts-api-version-keyw
     """
 
     def __init__(self, credential: AsyncTokenCredential, **kwargs: Any) -> None:
-        endpoint = kwargs.pop("endpoint", "https://api.loganalytics.io")
+        endpoint = kwargs.pop("endpoint", "https://api.loganalytics.io/v1")
         if not endpoint.startswith("https://") and not endpoint.startswith("http://"):
             endpoint = "https://" + endpoint
+        parsed_endpoint = urlparse(endpoint)
+        # Assume audience is the base URL of the endpoint, unless a value is explicitly passed in.
+        audience = kwargs.pop("audience", f"{parsed_endpoint.scheme}://{parsed_endpoint.netloc}")
         self._endpoint = endpoint
         auth_policy = kwargs.pop("authentication_policy", None)
         self._client = MonitorQueryClient(
             credential=credential,
-            authentication_policy=auth_policy or get_authentication_policy(credential, endpoint),
-            endpoint=self._endpoint.rstrip('/') + "/v1",
```

Comment:
curious, why did you remove the version number? @pvaneck

Reply:
The API endpoint includes `v1` and is defaulted to `https://api.loganalytics.io/v1` right above. If a user passes in a different endpoint, I'd rather not be appending additional things and just use what we get. So if a user wanted to use the Azure China Log Analytics endpoint, they'd explicitly pass in `endpoint="https://api.loganalytics.azure.cn/v1"`.